spec/protocol/addresses_keys/spend_key.md

```diff
@@ -0,0 +1,32 @@
+# Spend Keys
+
+The *spend key* has three components derived from the spend seed $ss$:
+
+* $ask$, the *spend authorization key*, a scalar value
+* $nsk$, the *nullifier private key*, a scalar value
+* $ovk$, the *outgoing viewing key*, a 32 byte number
+
+The scalars are derived by using the spend seed as the key for a PRF,
+expanding different labels, and converting the output to a `decaf377` scalar.
+In pseudocode this becomes:
+
+```
+prf_expand(x) = BLAKE2b_512("Penumbra_ExpandSeed", ss || x )
```

Comment:
I split this out because it's clearer to think of it as a PRF, separate from the details of the construction.